App.js

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import DeviceInfo from 'react-native-device-info';
 import FlashMessage from 'react-native-flash-message';
 import Toast from 'react-native-toast-message';
-import {BackHandler, KeyboardAvoidingView, Platform, View} from 'react-native';
+import {BackHandler, Platform, View} from 'react-native';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `KeyboardAvoidingView` has been removed. This is consistent with the changes in `root-stack.js` where the `KeyboardAvoidingView` component is now being used.
```diff
- import {BackHandler, KeyboardAvoidingView, Platform, View} from 'react-native';
+ import {BackHandler, Platform, View} from 'react-native';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->